app/src/main/java/com/teampophory/pophory/feature/home/store/apdater/StoreAdapter.kt

```diff
@@ -8,15 +8,21 @@ import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
 import com.teampophory.pophory.databinding.ItemStorePagerBinding
 
-class StoreAdapter(private val onItemClicked: (Int) -> Unit) :
-    ListAdapter<Int, StoreAdapter.StoreViewHolder>(StoreDiffCallback) {
+class StoreAdapter(
+    private val onItemClicked: (Int) -> Unit,
+    diffCallback: DiffUtil.ItemCallback<Int>
+) : ListAdapter<Int, StoreAdapter.StoreViewHolder>(diffCallback) {
```

Comment:
```suggestion
    diffCallback: DiffUtil.ItemCallback<Int>,
    private val onItemClicked: (Int) -> Unit,
) : ListAdapter<Int, StoreAdapter.StoreViewHolder>(diffCallback) {
```

패러미터에 람다식이 들어간 웬만한 경우에는 trailing lambda를 만들 수 있게 해주세요. 그래야 줄 인덴팅도 좀 덜고 가독성도 더 올라갑니다!

Reply:
네 감사합니당 ㅎㅎ